hack/download-hack-scripts.sh

```diff
@@ -24,8 +24,16 @@ ABS_DEST_DIR="${DEST_REPO}/${DEST_DIR}"
 # Delete existing kiali folder before download hack scripts
 rm -rf ${ABS_DEST_DIR}/{*,.[!.]*}
 
+# Get current OSSMC git branch to clone corresponding branch of Kiali (both applications should have the same version)
+KIALI_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [ "${KIALI_BRANCH}" == "main" ]; then
+    KIALI_BRANCH="v1.73"
```

Comment:
I think this will backfire once we will need multiple release branches or just bump OSSMC to different version than v1.73

Reply:
For the moment, the main branch of OSSMC works with Kiali 1.73. When we merge PR https://github.com/kiali/openshift-servicemesh-plugin/pull/278 to migrate OSSMC to PF5 we will have to change that line, setting the value to `master`

Reply:
For the moment, the main branch of OSSMC works with Kiali 1.73. When we merge PR https://github.com/kiali/openshift-servicemesh-plugin/pull/278 to migrate OSSMC to PF5 we will have to change that line, setting the value to `master`